### ISSUE TITLE
setting up a contact form on the help page 

### ISSUE THREAD

**Issue by sumukhshetty** [qjflores/exchangemvpreact#36]

**Comment by joshpitzalis** [comment]
this component has been built in pull req 30 https://github.com/qjflores/exchangemvpreact/pull/30. not connected though. How do you want to handle submissions?

**Comment by qjflores** [comment]
@sumukhshetty are we going to use mailgun for this?

**Comment by sumukhshetty** [comment]
yup for any email to be sent from our site, we have to use mailguns

**Comment by qjflores** [comment]
https://nodemailer.com/about/

I think we can use this specifically for users contacting us about help stuff. It's easy to implement, mailgun and other email solutions will be needed when we're concerned about our emails ending up in the user's spam email box - something like notification about trades. 

@sumukhshetty We'd need to create a gmail account like automtelabs@gmail.com. I can't do it here from Shenzen 

we'd add the password to our secrets folder

**Comment by qjflores** [comment]
@all implementing nodemailer exposes the gmail account - don't use this email account for anything sensitive like creating accounts on behave of the company

i'll try implementing this as a short term work around until we implement something like activecampaign

**Comment by qjflores** [comment]
looks like need to give up on nodemailer - it requires a nodejs server

this hack looks very promising though

https://medium.com/front-end-hacking/send-emails-from-firebase-with-zapier-15faeda36895

**Comment by qjflores** [comment]
the more I look into zapier webhooks the more excited i get about using them

https://zapier.com/blog/how-use-zapier-webhooks/

**Comment by qjflores** [comment]
it can integrate with mailgun!

https://www.youtube.com/watch?v=EKr6EJKkWEM

**Comment by adklempner** [comment]
Have you checked out firebase cloud functions? [https://firebase.google.com/docs/functions/](https://firebase.google.com/docs/functions/)

If any of this requires back-end processing cloud functions are perfect for that.

**Comment by qjflores** [comment]
@adklempner I'll take a look, this looks interesting too.

**Comment by qjflores** [comment]
https://github.com/firebase/functions-samples/tree/master/quickstarts/email-users
